magpie/run/magpie-run-project-zeppelin

```diff
@@ -82,9 +82,9 @@ Magpie_run_start_zeppelin () {
 
             # My rough estimate for setup time is 30 seconds
             local sleepwait=30
-
```

Comment:
accidental changes here?  no need?

Reply:
Funny...yes and no.  Prior to this week it would have been accidental changes.  Brian and I discovered a confilct with the .jar files in Zeppelin, so now this file has also been updated.